questions/Tracking Income

```diff
@@ -0,0 +1,34 @@
+#Tracking Income
+
+## Question
+
+Welcome to **Tracking Income**! Here is the problem statement:
+
+You are given a list of revenue and expenses incurred by your company. Revenue is money coming in, Expensises is money going out. Your job is to write a code to calculate how much money the company has at the end of the day.
```

Comment:
You could make more of a story here. Like give a scenario where its like "You are the accountant of a big company are you are trying to track money in and money out to make sure the company is not overspending. Here is a balance sheet of the day. Figure out if the company is making money,